test/recipes/30-test_evp_data/evpkdf_snmp.txt

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# Tests start with one of these keywords
+#       Cipher Decrypt Derive Digest Encoding KDF MAC PBE
+#       PrivPubKeyPair Sign Verify VerifyRecover
+# and continue until a blank line. Lines starting with a pound sign are ignored.
+
+Title = SNMPKDF tests (from NIST ACVP 1.0 test vectors)
+
+KDF = SNMPKDF
```

Comment:
You may need to add the following line before each KDF = SNMPKDF
FIPSversion = >=3.3.0


There are backwards compatibility tests for old versions of OpenSSL FIPS providers.
This line indicates only to test against new FIPS providers.

Reply:
done